libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1731,7 +1731,7 @@ bool TypeChecker::visit(UnaryOperation const& _operation)
 	TypeResult result = type(_operation.subExpression())->unaryOperatorResult(op);
 	if (!result)
 	{
-		string description = "Unary operator " + string(TokenTraits::toString(op)) + " cannot be applied to type " + subExprType->humanReadableName() + "." + (!result.message().empty() ? " " + result.message() : "");
+		string description = "Built-in unary operator " + string(TokenTraits::toString(op)) + " cannot be applied to type " + subExprType->humanReadableName() + "." + (!result.message().empty() ? " " + result.message() : "");
```

Comment:
```suggestion
		string description =
			fmt::format(
			"Built-in unary operator {} cannot be applied to type {}.{}",
			TokenTraits::toString(op),
			subExprType->humanReadableName(),
			(!result.message().empty() ? " " + result.message() : "")
			);
```
Breaking long line and we have been suggesting using `fmt::format` for string concatenations like this. Maybe we could also do that for the other error messages modified by this PR?

Reply:
@matheusaaguiar your formatting is a bit off, but I agree with this :D

Reply:
Ah, sometimes it is hard to get it right in the comments/suggestions :)

Reply:
Ok, why not. I don't think we should necessarily jump in to convert everything to fmtlib, but I do find that more readable than adding strings so I can do it here.